pavelib/bok_choy.py

```diff
@@ -32,7 +29,31 @@
     make_option("-v", "--verbosity", action="count", dest="verbosity"),
     make_option("--pdb", action="store_true", help="Drop into debugger on failures or errors"),
     make_option("--skip_firefox_version_validation", action='store_false', dest="validate_firefox_version")
-])
+]
+
+
+def parse_bokchoy_opts(options):
```

Comment:
Ah sorry I just added a new opt for multiprocessing. You'll want to add that in here when you rebase.